data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -67,6 +69,74 @@ then
 	touch cvo-bootstrap.done
 fi
 
+# We originally wanted to run the etcd cert signer as
+# a static pod, but kubelet could't remove static pod
+# when API server is not up, so we have to run this as
+# podman container.
+# See https://github.com/kubernetes/kubernetes/issues/43292
+
+echo "Starting etcd certificate signer..."
+
+trap "podman rm --force etcd-signer" ERR
+
+bootkube_podman_run \
+        --name etcd-signer \
+        --detach \
+        --volume /opt/openshift/tls:/opt/openshift/tls:ro,z \
+        "${KUBE_ETCD_SIGNER_SERVER_IMAGE}" \
+        serve \
+        --cacrt=/opt/openshift/tls/etcd-signer.crt \
+        --cakey=/opt/openshift/tls/etcd-signer.key \
+        --metric-cacrt=/opt/openshift/tls/etcd-metric-signer.crt \
+        --metric-cakey=/opt/openshift/tls/etcd-metric-signer.key \
+        --servcrt=/opt/openshift/tls/kube-apiserver-lb-server.crt \
+        --servkey=/opt/openshift/tls/kube-apiserver-lb-server.key \
+        --servcrt=/opt/openshift/tls/kube-apiserver-internal-lb-server.crt \
+        --servkey=/opt/openshift/tls/kube-apiserver-internal-lb-server.key \
+        --servcrt=/opt/openshift/tls/kube-apiserver-localhost-server.crt \
+        --servkey=/opt/openshift/tls/kube-apiserver-localhost-server.key \
+        --address=0.0.0.0:6443 \
+        --insecure-health-check-address=0.0.0.0:6080 \
+        --csrdir=/tmp \
+        --peercertdur=26280h \
+        --servercertdur=26280h \
+        --metriccertdur=26280h
+
+
+if [ ! -f etcd-bootstrap.done ]
+then
+       echo "Rendering etcd Operator Manifests..."
+       rm -rf etcd-bootstrap
+       mkdir -p /etc/etcd
+       bootkube_podman_run \
+           --quiet \
+           --volume "$PWD:/assets:z" \
+           --volume /etc/kubernetes:/etc/kubernetes:z \
+           --volume /etc/etcd:/etc/etcd:z \
+           "${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+           /usr/bin/cluster-etcd-operator render \
+           --etcd-ca=/assets/tls/etcd-ca-bundle.crt \
+           --etcd-metric-ca=/assets/tls/etcd-metric-ca-bundle.crt \
+           --manifest-etcd-image "${MACHINE_CONFIG_ETCD_IMAGE}" \
+           --etcd-discovery-domain {{.ClusterDomain}} \
+           --manifest-cluster-etcd-operator-image "${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+           --manifest-setup-etcd-env-image "${MACHINE_CONFIG_OPERATOR_IMAGE}" \
+           --manifest-image-pull-policy "TEST" \
+           --manifest-kube-client-agent-image "${MACHINE_CONFIG_KUBE_CLIENT_AGENT_IMAGE}" \
+           --asset-input-dir /assets/tls \
+           --asset-output-dir /assets/etcd-bootstrap \
+           --config-output-file /assets/etcd-bootstrap/config
+
+       cp etcd-bootstrap/manifests/* manifests/
+       cp etcd-bootstrap/bootstrap-manifests/etcd-member-pod.yaml /etc/kubernetes/manifests/
+       touch etcd-bootstrap.done
+fi
+
+if [ -f /etc/kubernetes/manifests/etcd-member-pod.yaml ]
+then
+	ETCD_ENDPOINTS=$(printf "https://etcd-bootstrap.%s:2379" "{{.ClusterDomain}}")
```

Comment:
nit: Why not just use `ETCD_ENDPOINTS="https://etcd-bootstrap.{{.ClusterDomain}}:2379"`?